File: simpleApi/src/main/java/com/lifemmo/pl/simple/api/handler/DisconnectApi.java
package com.lifemmo.pl.simple.api.handler;

import com.lifemmo.pl.base.service.api.ApiCallBackHandler;
import lombok.extern.slf4j.Slf4j;
import org.jetbrains.annotations.NotNull;

@Slf4j
public class DisconnectApi implements ApiCallBackHandler {

    @Override
    public void onDisconnect(long accountId, @NotNull String sessionInfo){
        log.info("onDisconnect {},{}",accountId,sessionInfo);
    }
}

File: simpleApi/src/main/kotlin/org/ulalax/playhouse/simple/api/handler/DisconnectApi.kt
package org.ulalax.playhouse.simple.api.handler

import org.apache.logging.log4j.kotlin.logger
import org.ulalax.playhouse.service.api.ApiCallBack


class DisconnectApi : ApiCallBack {
    private val log = logger()

    override fun onDisconnect(accountId: Long) {
        log.info("onDisconnect $accountId")
    }
}
